local/bin/py/build/actions/workflows.py

```diff
@@ -41,7 +41,7 @@ def workflows(content, content_dir):
                 p = Path(file_name)
                 for action_name, action_data in data.get('actions', {}).items():
                     action_stability = action_data.get('stability')
-                    if not action_stability or action_stability == 'stable':
+                    if not action_stability or action_stability == 'stable' and not data.get('internal') and not data.get('hidden') and not data.get('deprecated'):
```

Comment:
@dd-gplassard in your in-progress docs, you define a `visibility`  field instead of `internal`. should this be accounted for as well?

Reply:
Yes sorry about that, actually in our code we use a `@visible false` annotation but it gets translated into a `"hidden": true` in our manifest :/
So the check should be on hidden

Code annotation : https://github.com/DataDog/dd-source/blob/5f528e423c17084c4e9e1eec907f8b40a9959901/domains/workflow/actionplatform/apps/wf-actions-worker/src/runner/bundles/com.datadoghq.internal/actions.ts#L34
Manifest field : https://github.com/DataDog/dd-source/blob/5f528e423c17084c4e9e1eec907f8b40a9959901/domains/workflow/actionplatform/apps/wf-actions-worker/src/runner/bundles/com.datadoghq.internal/manifest.json#L222

And on a side note we might be able to remove this concept so maybe a cleanup PR will follow shortly :)